contrib/dockerd-rootless-setuptool.sh

```diff
@@ -307,6 +307,7 @@ install_systemd() {
 			TasksMax=infinity
 			Delegate=yes
 			Type=simple
+			KillMode=mixed
```

Comment:
Does/should rootless mode support "live-restore"? ISTR we use "process" in the regular systemd unit because of that; https://github.com/docker/docker-ce-packaging/blob/master/systemd/docker.service#L43

Reply:
No, at least for now

Reply:
"process" does seem more correct eventually, but mixed sounds good for now :+1: